src/shared/constants/constant.ts

```diff
@@ -0,0 +1,16 @@
+const logInInitialValues = {
+    username: '',
+    password: '',
+};
+
+const signUpInitialValues = {
```

Comment:
you can use destructure **logInInitialValues** in **signUpInitialValues**

Reply:
Done